packages/go_router/lib/src/parser.dart

```diff
@@ -2,22 +2,20 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: use_build_context_synchronously
```

Comment:
we should not do ignore for file

Reply:
whenever this lint shows up, try to check for `context.isMounted` before continue the operation